src/components/common/AudioLibrarySearchField.vue

```diff
@@ -3,6 +3,7 @@
     <div class="input-group">
       <span class="input-group-prepend">
         <button :class="{active: isEntireLibraryModeEnabled}"
+                :disabled="!isEditable"
```

Comment:
This is good for a 1st PR in flow-builder.
I noticed you introduced `isEditdable` prop in many components to disable the side bar children.
I would suggest we take advantage of the `<fieldset>` like here:  https://bitbucket.org/voto/voto5/src/e642ee81827f72c654cced81c9e304e80e4d64ea/resources/assets/js/trees/components/block-types/ValidateCodeBlock.vue#lines-10

This way the maintenance will be less painful because we don't need to add `isEditable` for future block config elements. 

Here is a overview how to disable it
- change the element having `tree-sidebar` class into `<fieldset>`
- put conditional disabled there
<img width="950" alt="Screen Shot 2021-02-23 at 1 09 39 PM" src="https://user-images.githubusercontent.com/68745918/108896092-1163b700-75da-11eb-87ee-835ee4607fe4.png">

Reply:
But pls get confirmation from @bzabos first if he is good with this suggestion

Reply:
@safydy-r  Thanks for your suggestions. I will try out fieldset and update the PR.